src/shop-client.js

```diff
@@ -142,7 +142,7 @@ const ShopClient = CoreObject.extend({
    * ```
    *
    * @method fetchQuery
-   * @public
+   * @private
```

Comment:
in Slate we're marking private methods as private for the purpose of hiding them from the docs... But IMO the 2 should go hand in hand.  Same with `protected` -- because both generally has meaning in terms of OOP, I'm hesitant to use either for presentation purposes. 

Is there a specific reason you guys want to hide a public method from the docs (maybe an alternative is simply don't doc-block this method?)?